keras_nlp/models/task.py

```diff
@@ -191,7 +191,16 @@ def from_preset(
                 preset,
                 config_file="tokenizer.json",
             )
-            preprocessor = cls.preprocessor_cls(tokenizer=tokenizer)
+            if "backbone" in kwargs:
+                raise ValueError(
+                    "`backbone` weights are loaded from the "
```

Comment:
I would reword this so we give the user an action they can do to remedy this, which is pretty simple they should call the default constructor and not `from_preset`. Something like...

```python
"You cannot pass a `backbone` argument to the `from_preset` method. "
f"Instead, call the {cls.__name__} default constructor with `backbone` argument. "
f"Received: backbone={backbone}"
```

Reply:
Thanks for rewording it. Done.